dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/DualScreenInfo.kt

```diff
@@ -56,29 +57,41 @@ class DualScreenInfo constructor(context: ReactApplicationContext) : ReactContex
 			}
 			return statusBarHeight;
 		}
+
+	private val mNavigationBarHeight: Int
+		get() {
+			val rectangle = Rect()
+			val displayMetrics = DisplayMetrics()
+			currentActivity?.window?.decorView?.getWindowVisibleDisplayFrame(rectangle)
+			currentActivity?.windowManager?.defaultDisplay?.getRealMetrics(displayMetrics);
+			return displayMetrics.heightPixels - (rectangle.top + rectangle.height());
+		}
+
 	private val windowRects: List<Rect>
 		get() {
 			val boundings = mDisplayMask?.getBoundingRectsForRotation(rotation)
+			val barHeights = mStatusBarHeight + mNavigationBarHeight
 			val windowBounds = windowRect;
 			return if (boundings == null || boundings.size == 0) {
-				windowBounds.bottom = windowBounds.bottom - mStatusBarHeight;
+				windowBounds.bottom = windowBounds.bottom - barHeights;
 				listOf(windowBounds)
 			} else {
 				val hingeRect = boundings[0]
 				if (hingeRect.top == 0) {
-					windowBounds.bottom = windowBounds.bottom - mStatusBarHeight;
+					windowBounds.bottom = windowBounds.bottom - barHeights;
 					val leftRect = Rect(0, 0, hingeRect.left, windowBounds.bottom)
 					val rightRect = Rect(hingeRect.right, 0, windowBounds.right, windowBounds.bottom)
 					listOf(leftRect, rightRect)
 				} else {
-					hingeRect.bottom = hingeRect.bottom - mStatusBarHeight;
+					hingeRect.bottom = hingeRect.bottom - barHeights;
```

Comment:
>hingeRect.bottom - barHeights [](start = 24, length = 29)

This doesn't look right.  This case is handling the scenario where you've rotated the device and the screens are top/bottom.  In this case, the hinge rect, which splits the screens, should move its bottom to account for the status bar, but not the nav bar, because the nav bar is below the hinge.  

Can you please test this scenario and confirm?